src/auto-evo/steps/ModifyExistingSpecies.cs

```diff
@@ -205,7 +206,7 @@ public bool RunStep(RunResults results)
                         var oldScore = cache.GetPressureScore(pressure, patch, species);
 
                         // Break if mutation fails a pressure
-                        if (newScore <= 0)
+                        if (newScore < 0)
```

Comment:
Are you sure this should be less than zero? I recall the logic being strictly positive before...

Reply:
I think I will revert that. I think I misremembered two species that score the same being 0 instead of 1.